README.md

```diff
@@ -1,15 +1,17 @@
 # Vue Function API
 
-> [通过基于函数的 API 来复用组件逻辑](https://zhuanlan.zhihu.com/p/68477600)
+> [Function-based Component API RFC](https://github.com/vuejs/rfcs/pull/42)
```

Comment:
The RFC link should be https://github.com/vuejs/rfcs/blob/function-apis/active-rfcs/0000-function-api.md

Reply:
Fixed